backbone.js

```diff
@@ -1455,6 +1455,16 @@
 
   // Set up all inheritable **Backbone.Router** properties and methods.
   _.extend(Router.prototype, Events, {
+    // If set to true, route handlers will be called with routeData object
+    //
+    //     this.route('search/:query/p:num', 'search', function(routeData) {
```

Comment:
:+1: to `routeData` over `1...n` args. I'd like to see Backbone's router give even more info in route callbacks, and a `1...n` arg API would get a little unwieldy with that many args.